techradar/radars/migrations/0002_alter_radar_slug.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 5.1.4 on 2025-01-03 07:47
```

Comment:
Jeśli nie ma takiej konieczności (a najczęściej nie ma), to staraj się unikać dodawania więcej niż jednej migracji per appka w jednym PR. Jako, że to dopiero pierwsze migracje w całym projekcie, to możesz po prostu usunąć swój plik bazy sqlite, usunąć te dwa pliki i stworzyć od nowa plik 0001 (`python manage.py makemigrations`).

Hint: jak będziesz zmieniać argument `populate_from` to django też będzie chciało dodać migrację, warto tamtą zmianę zrobić wcześniej :slightly_smiling_face:

Reply:
można tego nie robić, ale wtedy liczba migracji bardzo szybko rośnie, co z czasem może utrudnić utrzymanie projektu :smile: